2021-05-28 07:42:14,534 INFO    MainThread:29752 [wandb_setup.py:_flush():69] setting env: {}
2021-05-28 07:42:14,534 INFO    MainThread:29752 [wandb_setup.py:_flush():69] setting login settings: {}
2021-05-28 07:42:14,534 INFO    MainThread:29752 [wandb_init.py:_log_setup():336] Logging user logs to /opt/ml/gitclone/p4-mod-model_diet/wandb/run-20210528_074214-1yatenud/logs/debug.log
2021-05-28 07:42:14,534 INFO    MainThread:29752 [wandb_init.py:_log_setup():337] Logging internal logs to /opt/ml/gitclone/p4-mod-model_diet/wandb/run-20210528_074214-1yatenud/logs/debug-internal.log
2021-05-28 07:42:14,534 INFO    MainThread:29752 [wandb_init.py:init():369] calling init triggers
2021-05-28 07:42:14,534 INFO    MainThread:29752 [wandb_init.py:init():374] wandb.init called with sweep_config: {}
config: {'img_size': 224, 'batch_size': 64, 'init_lr': 0.00043474822991771203}
2021-05-28 07:42:14,534 INFO    MainThread:29752 [wandb_init.py:init():389] re-initializing run, found existing run on stack: 1d1ii72e
2021-05-28 07:42:14,534 INFO    MainThread:29752 [wandb_run.py:finish():1208] finishing run seokjin/optuna/1d1ii72e
2021-05-28 07:42:14,534 INFO    MainThread:29752 [wandb_run.py:_atexit_cleanup():1535] got exitcode: 0
2021-05-28 07:42:14,535 INFO    MainThread:29752 [wandb_run.py:_restore():1507] restore
2021-05-28 07:42:16,746 INFO    MainThread:29752 [wandb_run.py:_wait_for_finish():1657] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 717
  total_bytes: 717
}

2021-05-28 07:42:17,356 INFO    MainThread:29752 [wandb_run.py:_wait_for_finish():1657] got exit ret: file_counts {
  wandb_count: 4
}
pusher_stats {
  uploaded_bytes: 717
  total_bytes: 4032
}

2021-05-28 07:42:17,744 INFO    MainThread:29752 [wandb_run.py:_wait_for_finish():1657] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 1371
  total_bytes: 4227
}

2021-05-28 07:42:17,846 INFO    MainThread:29752 [wandb_run.py:_wait_for_finish():1657] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 1566
  total_bytes: 4227
}

2021-05-28 07:42:17,947 INFO    MainThread:29752 [wandb_run.py:_wait_for_finish():1657] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 1566
  total_bytes: 4227
}

2021-05-28 07:42:18,048 INFO    MainThread:29752 [wandb_run.py:_wait_for_finish():1657] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 4227
  total_bytes: 4227
}

2021-05-28 07:42:18,150 INFO    MainThread:29752 [wandb_run.py:_wait_for_finish():1657] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 4227
  total_bytes: 4227
}

2021-05-28 07:42:18,251 INFO    MainThread:29752 [wandb_run.py:_wait_for_finish():1657] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 4227
  total_bytes: 4227
}

2021-05-28 07:42:18,353 INFO    MainThread:29752 [wandb_run.py:_wait_for_finish():1657] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 4227
  total_bytes: 4227
}

2021-05-28 07:42:18,454 INFO    MainThread:29752 [wandb_run.py:_wait_for_finish():1657] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 4227
  total_bytes: 4227
}

2021-05-28 07:42:18,555 INFO    MainThread:29752 [wandb_run.py:_wait_for_finish():1657] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 4227
  total_bytes: 4227
}

2021-05-28 07:42:18,656 INFO    MainThread:29752 [wandb_run.py:_wait_for_finish():1657] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 4227
  total_bytes: 4227
}

2021-05-28 07:42:19,743 INFO    MainThread:29752 [wandb_run.py:_show_summary():1812] rendering summary
2021-05-28 07:42:19,743 INFO    MainThread:29752 [wandb_run.py:_show_history():1850] rendering history
2021-05-28 07:42:19,743 INFO    MainThread:29752 [wandb_run.py:_show_files():1879] logging synced files
2021-05-28 07:42:19,746 INFO    MainThread:29752 [wandb_init.py:init():418] starting backend
2021-05-28 07:42:19,746 INFO    MainThread:29752 [backend.py:_multiprocessing_setup():70] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2021-05-28 07:42:19,749 INFO    MainThread:29752 [backend.py:ensure_launched():132] starting backend process...
2021-05-28 07:42:19,803 INFO    MainThread:29752 [backend.py:ensure_launched():136] started backend process with pid: 929
2021-05-28 07:42:19,805 INFO    MainThread:29752 [wandb_init.py:init():423] backend started and connected
2021-05-28 07:42:19,807 INFO    MainThread:29752 [wandb_init.py:init():465] updated telemetry
2021-05-28 07:42:19,808 INFO    MainThread:29752 [wandb_init.py:init():484] communicating current version
2021-05-28 07:42:20,567 INFO    MainThread:29752 [wandb_init.py:init():489] got version response upgrade_message: "wandb version 0.10.31 is available!  To upgrade, please run:\n $ pip install wandb --upgrade"

2021-05-28 07:42:20,568 INFO    MainThread:29752 [wandb_init.py:init():497] communicating run to backend with 30 second timeout
2021-05-28 07:42:20,795 INFO    MainThread:29752 [wandb_init.py:init():522] starting run threads in backend
2021-05-28 07:42:20,833 INFO    MainThread:29752 [wandb_run.py:_console_start():1565] atexit reg
2021-05-28 07:42:20,834 INFO    MainThread:29752 [wandb_run.py:_redirect():1412] redirect: SettingsConsole.REDIRECT
2021-05-28 07:42:20,834 INFO    MainThread:29752 [wandb_run.py:_redirect():1417] Redirecting console.
2021-05-28 07:42:20,835 INFO    MainThread:29752 [wandb_run.py:_redirect():1479] Redirects installed.
2021-05-28 07:42:20,835 INFO    MainThread:29752 [wandb_init.py:init():546] run started, returning control to user process
2021-05-28 07:53:20,622 INFO    MainThread:29752 [wandb_setup.py:_flush():69] setting env: {}
2021-05-28 07:53:20,622 INFO    MainThread:29752 [wandb_setup.py:_flush():69] setting login settings: {}
2021-05-28 07:53:20,623 INFO    MainThread:29752 [wandb_init.py:_log_setup():336] Logging user logs to /opt/ml/gitclone/p4-mod-model_diet/wandb/run-20210528_075320-3b5kk0ka/logs/debug.log
2021-05-28 07:53:20,623 INFO    MainThread:29752 [wandb_init.py:_log_setup():337] Logging internal logs to /opt/ml/gitclone/p4-mod-model_diet/wandb/run-20210528_075320-3b5kk0ka/logs/debug-internal.log
2021-05-28 07:53:20,623 INFO    MainThread:29752 [wandb_init.py:init():369] calling init triggers
2021-05-28 07:53:20,623 INFO    MainThread:29752 [wandb_init.py:init():374] wandb.init called with sweep_config: {}
config: {'img_size': 192, 'batch_size': 32, 'init_lr': 0.0002700339862043518}
2021-05-28 07:53:20,623 INFO    MainThread:29752 [wandb_init.py:init():389] re-initializing run, found existing run on stack: 1yatenud
2021-05-28 07:53:20,623 INFO    MainThread:29752 [wandb_run.py:finish():1208] finishing run seokjin/optuna/1yatenud
